README.md

```diff
@@ -116,7 +116,7 @@ pub fn shave_the_yak(yak: &mut Yak) {
                 break;
             }
             Err(err) => {
-                warn!(err:error = err; "Unable to locate a razor, retrying");
+                warn!(err:err; "Unable to locate a razor, retrying");
```

Comment:
Nit: maybe this makes the example a little confusing? Making it look like `:` is the separator for key-value pairs?

Reply:
That's a good point. It could also look like it's trying to be a pattern binding 🤔 I'll try to clear it up.